buildSrc/src/main/kotlin/otel.japicmp-conventions.gradle.kts

```diff
@@ -42,35 +42,38 @@ fun findArtifact(version: String): File {
     }
 }
 
-afterEvaluate {
-    tasks {
-        val jApiCmp by registering(JapicmpTask::class) {
-            dependsOn("jar")
-            // the japicmp "old" version is either the user-specified one, or the latest release.
-            val apiBaseVersion: String? by project
-            val baselineVersion = apiBaseVersion ?: latestReleasedVersion
-            val baselineArtifact = findArtifact(baselineVersion)
-            oldClasspath = files(baselineArtifact)
+// generate the api diff report for any module that is stable and publishes a jar.
+if (!project.hasProperty("otel.release") && !project.name.startsWith("bom")) {
```

Comment:
This seems like a "feature" of the japicmp conventions, not publishing ones so didn't use the conditional plugin application like before